docs/directives/v-always-look-at.md

```diff
@@ -2,6 +2,10 @@
 
 With the new directive v-always-look-at provided by **TresJS**, you can add easily command an [Object3D](https://threejs.org/docs/index.html?q=object#api/en/core/Object3D) to always look at a specific position, this could be passed as a Vector3 or an Array.
 
+::: warning
+This directive can be only be used inside of a `TresCanvas` since this component acts as the provider for the context data and it uses `onLoop` under the hood.
```

Comment:
Does this work? I'm getting

```
Error: useTresContext must be used together with useTresContextProvider
```

```vue
<script setup lang="ts">
import { shallowRef } from 'vue'
import { vAlwaysLookAt, vLightHelper } from '@tresjs/core'

const other = shallowRef([1,1,1])

setTimeout(() => other.value = [2,1,1], 1000)
</script>

<template>
  <TresDirectionalLight v-light-helper v-always-look-at="other" :position="[3, 3, 3]" :intensity="1" />
</template>
```

Reply:
I'm gonna remove the directives with loop. They are not adding enough value for the troubles